count.rb

```diff
@@ -0,0 +1,40 @@
+class WordCount
+  attr_accessor :speech
+  attr_reader :counted_words
+
+  def load_file(path)
+    @speech = File.read(path)
+  end
+
+  def parse_words
+    @speech.gsub!(/[^0-9a-zA-Z\s]/, '')
+    @speech.gsub!(/\s+/, ' ')
+    @speech.downcase!.strip!
+    @speech = @speech.split(' ')
+  end
+
+  def count_words
+    @counted_words = Hash.new
+    @speech.each do |word|
+      if @counted_words.include?(word)
+        @counted_words[word] += 1
+      else
+        @counted_words[word] = 1
+      end
+    end
+  end
```

Comment:
Take a look at inject method, you can start with a Hash.new(0) to store the count pairs. Check the argument on the Hash constructor, it returns that default when you try to access a new key. http://docs.ruby-lang.org/en/2.0.0/Hash.html#method-c-new